scalding-commons/src/main/scala/com/twitter/scalding/commons/source/LzoGenericScheme.scala

```diff
@@ -55,21 +55,27 @@ private[source] object ExternalizerSerializer {
   }
 }
 
-private[source] object ConfigBinaryConverterProvider {
-  val ProviderConfKey = "com.twitter.scalding.lzo.converter.provider"
+private[source] object SourceConfigBinaryConverterProvider {
```

Comment:
I don't think you need this object, nor SinkConfigBinaryConverterProvider

just put these vals in:
```
object ConfigBinaryConverterProvider {
  val sourceKey = "com.twitter.scalding.lzo.converter.provider.source"
  val sinkKey = "com.twitter.scalding.lzo.converter.provider.sink"
}
```